append-to-bash.py

```diff
@@ -0,0 +1,19 @@
+import json
```

Comment:
@joeltejeda please may you also name this `update-WP.py` because 'append-to-bash' doesn't give much clue